### ISSUE TITLE
Failed to deploy when using tags and wafConfig

### ISSUE THREAD

**Issue by WolfWalter** [sid88in/serverless-appsync-plugin#443]
When I try to deploy the api with 
```
{
  wafConfig: { enabled: true, rules: [{ throttle: 300 }] },
  tags: {
    test: 'test'
  }
...
}
```
the deployment fails with the folowing error:

```
Serverless Error ----------------------------------------
 
  An error occurred: GraphQlWaf - Resource handler returned message: "Model validation failed (#/Rules/0/Priority: expected type: Number, found: String
  #/Rules/0/Statement/RateBasedStatement/Limit: expected type: Number, found: String
  #/Rules/0/VisibilityConfig/SampledRequestsEnabled: expected type: Boolean, found: String
  #/Rules/0/VisibilityConfig/CloudWatchMetricsEnabled: expected type: Boolean, found: String
  #/VisibilityConfig/SampledRequestsEnabled: expected type: Boolean, found: String
  #/VisibilityConfig/CloudWatchMetricsEnabled: expected type: Boolean, found: String
  #/Tags: expected type: JSONArray, found: JSONObject)" (RequestToken: 9b518598-fcd9-5f03-7edd-62d94d2a35a7, HandlerErrorCode: InvalidRequest).
```

The `cloudformation-template-update-stack.json` contains the following WAF-Cofnig:
```
...
 "GraphQlWaf": {
      "Type": "AWS::WAFv2::WebACL",
      "Properties": {
        "DefaultAction": {
          "Allow": {}
        },
        "Scope": "REGIONAL",
        "Description": "ACL rules for AppSync...",
        "Name": "sp-api-xxxWaf",
        "Rules": [
          {
            "Action": {
              "Block": {}
            },
            "Name": "BaseThrottle",
            "Priority": 100,
            "Statement": {
              "RateBasedStatement": {
                "AggregateKeyType": "IP",
                "Limit": 300
              }
            },
            "VisibilityConfig": {
              "CloudWatchMetricsEnabled": true,
              "MetricName": "BaseThrottle",
              "SampledRequestsEnabled": true
            }
          }
        ],
        "VisibilityConfig": {
          "CloudWatchMetricsEnabled": true,
          "MetricName": "sp-api-xxxWaf",
          "SampledRequestsEnabled": true
        },
        "Tags": {
          "test": "test"
        }
      }
    },
...
```

But for [`AWS::WAFv2::WebACL` ](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-wafv2-webacl.html) it should be a list of:

```
{
  "Key" : String,
  "Value" : String
}
```

**Comment by aschokking** [comment]
Thanks for the quick fix @WolfWalter, I was running into this now as well.

@bboure what's the process for publishing a new version to npm that would have this fix in it? 🤞

**Comment by bboure** [comment]
I will push a new version :)

**Comment by bboure** [comment]
Published in 1.21.1  🎉 

Thank you All
